en/sensor/optical_flow.md

```diff
@@ -0,0 +1,51 @@
+# Optical Flow
+
+Optical Flow uses a downward facing camera and a downward facing distance sensor for position estimation. 
+Optical Flow based navigation is supported by: EKF2 and LPE.
+
+## Setup
+
+An Optical Flow setup requires a downward facing camera which publishes to the [OPTICAL_FLOW_RAD](https://mavlink.io/en/messages/common.html#OPTICAL_FLOW_RAD) topic and a [distance sensor](../sensor/rangefinders.md) (preferably a LiDAR) publishing messages to the [DISANCE_SENSOR](https://mavlink.io/en/messages/common.html#DISTANCE_SENSOR) topic.
+
+The output of the flow has to be as follows
+
+| Vehicle movement | Integrated flow |
```

Comment:
I did not know what this meant - ie @bkueng Is this something that someone has to do? Where is the X and Y set - is it in the OPTICAL_FLOW_RAD?